src/_pytest/mark/structures.py

```diff
@@ -473,14 +472,13 @@ def test_function():
 
     # See TYPE_CHECKING above.
     if TYPE_CHECKING:
-        # Using casts instead of type comments intentionally - issue #7473.
         # TODO(py36): Change to builtin annotation syntax.
-        skip = cast(_SkipMarkDecorator, None)
-        skipif = cast(_SkipifMarkDecorator, None)
-        xfail = cast(_XfailMarkDecorator, None)
-        parametrize = cast(_ParametrizeMarkDecorator, None)
-        usefixtures = cast(_UsefixturesMarkDecorator, None)
-        filterwarnings = cast(_FilterwarningsMarkDecorator, None)
+        skip = _SkipMarkDecorator(Mark("skip", (), {}))
+        skipif = _SkipifMarkDecorator(Mark("skipif", (), {}))
+        xfail = _XfailMarkDecorator(Mark("xfail", (), {}))
+        parametrize = _ParametrizeMarkDecorator(Mark("parametrize ", (), {}))
+        usefixtures = _UsefixturesMarkDecorator(Mark("usefixtures ", (), {}))
+        filterwarnings = _FilterwarningsMarkDecorator(Mark("filterwarnings ", (), {}))
```

Comment:
```suggestion
        parametrize = _ParametrizeMarkDecorator(Mark("parametrize", (), {}))
        usefixtures = _UsefixturesMarkDecorator(Mark("usefixtures", (), {}))
        filterwarnings = _FilterwarningsMarkDecorator(Mark("filterwarnings", (), {}))
```

Reply:
probably worth fixing this before the backport

Reply:
Thanks, I am going to fix in master and add the backport of *that* as well to this PR.